assets/src/editor/events/tickets/editor-ticket/grid-view/inline-edit-ticket-price.js

```diff
@@ -37,11 +38,16 @@ const InlineEditTicketPrice = ( {
 	);
 	const ticketPriceAmount = ticket.price.amount.toNumber();
 
+	const basePrice = useBasePrice( prices );
 	const updateTicketPrice = useCallback(
 		( amount ) => {
 			amount = parseMoneyValue( amount );
 			ticket.price = new Money( amount, SiteCurrency );
-			calculateTicketBasePrice( amount );
+			const basePriceValue = calculateTicketBasePrice( amount );
+			basePrice.amount = new Money(
+				parseMoneyValue( basePriceValue ),
+				SiteCurrency
+			);
```

Comment:
I know this is a super small block of code,
but if we need to use this logic in more places
we should think about extracting it into a central location.
I'm thinking that mutations folder you added recently would be a good place for things like this,
but I also want to hold off until we talk about the whole model refactor thing because that could change everything

Reply:
Yes, we can do the same as we did for updating related tickets for a datetime. We can create a mutation function for such operation and we can use it wherever we need it. I think that can be done in another PR.